utils/scripts/format.sh

```diff
@@ -39,6 +39,6 @@ for f in $(git grep --untracked -ail res -- :/*.hpp :/*.cpp); do
     if [ ${VERBOSE} -ge 1 ]; then
        echo ${f}
     fi
-    ${CFM} -i ${REPO}/${f}
+    ${CFM} -style=file -i ${REPO}/${f}
```

Comment:
FYI - this change just brings the format script in line with what the CI is doing. I don't think its omission was _really_ messing anything up, but I just made the change as part of trying to figure out why my clang format wasn't working.